cmd/snap/cmd_get.go

```diff
@@ -257,17 +259,11 @@ func (x *cmdGet) Execute(args []string) error {
 	var conf map[string]interface{}
 	var err error
 	if isConfdbViewID(snapName) {
-		if err := validateConfdbFeatureFlag(); err != nil {
-			return err
-		}
-
 		// first argument is a confdbViewID, use the confdb API
-		confdbViewID := snapName
-		if err := validateConfdbViewID(confdbViewID); err != nil {
-			return err
+		if x.NoWait {
+			return errors.New("cannot use --no-wait when reading confdb")
```

Comment:
I don't think `--no-wait` makes sense for normal `x.client.Conf` call as well since it is already a sync call?